modules/indexing/src/test/java/org/apache/ignite/internal/processors/query/h2/twostep/GridMapQueryExecutorTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2025 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.query.h2.twostep;
+
+import org.apache.ignite.IgniteSystemProperties;
+import org.apache.ignite.internal.processors.cache.query.GridCacheSqlQuery;
+
+import org.apache.ignite.internal.util.IgniteUtils;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.UUID;
+
+import static org.junit.Assert.assertTrue;
+
+public class GridMapQueryExecutorTest {
+
+    public void testBuildQueryLogDetailsWithSampleParams(String sensitivity, String expectedParams) {
```

Comment:
It's better to not hardcode params value in the method body, but rather accept it as parameter, because it will make things much more obvious at the callsite. I.e. we pass following params and we expect to see following result.

P.S. I remember that It was me, who suggested such signature. Sorry if I confused you, but don't threat my code samples as perfect :smile: by default.